snap/snapcraft.yaml

```diff
@@ -34,6 +34,19 @@ apps:
     plugs: [home, removable-media, network]
     environment:
       HOME: $SNAP_USER_COMMON
+  tx:
+    command: bitcoin-tx
+    environment:
+      HOME: $SNAP_USER_COMMON
+  wallet:
+    command: bitcoin-wallet
+    plugs: [home, network]
```

Comment:
Can you explain why you have chosen those plugs? Also, why not `removable-media`?

Reply:
I was trying to use only the minimum amount of plugs necessary, but looking closely I think all of them use network and save/read a file, so it seems like it's better to use `home, removable-media, network` just like `bitcoin-cli`. I've changed the last commit adding these plugs.

Let me know what you think.

Reply:
I don't think bitcoin-tx/util needs any file or network access. wallet shouldn't need any network access?

Reply:
@MarcoFalke all 3 binaries have the `-signetseednode` option, so I supposed they need some network access:

```
-signetseednode
       Specify a seed node for the signet network, in the hostname[:port]
       format, e.g. sig.net:1234 (may be used multiple times to specify
       multiple seed nodes; defaults to the global default signet test
       network seed node(s))
```

Also, bitcoin-tx has this option so I supposed it needs access to home/removable-media to read a file:

```
load=NAME:FILENAME
       Load JSON file FILENAME into register NAME
```

And bitcoin-util doesn't look like it reads or writes any file, but have the `-signetseednode` like the others.

So maybe the correct plug should be:
wallet: `home, removable-media, network`
tx: `home, removable-media, network`
util: `network`

what do you think? :thinking:

Reply:
I think it is a bug for any of them to offer `signetseednode`.

Reply:
It makes sense if it's a bug, all of them have the same section `Chain selection options:`, which doesn't make sense any of them.

In this case, I removed the plugs from tx and util. Left only `home, removable-media` in wallet.